source/includes/fact-agg-pipeline-preview-toggle.rst

```diff
@@ -0,0 +1,3 @@
+The toggle to the right of the name of each pipeline stage dictates whether that 
+stage is included in the pipeline. Toggling a pipeline stage also updates the 
+pipeline preview, which reflects whether or not that stage is included. 
```

Comment:
Maybe simplify "which reflects whether or not..." to "which updates based on whether or not..."